tests/serving/test_serving.py

```diff
@@ -36,6 +37,16 @@
 from mlrun.utils import logger
 
 
+def setUpModule():
+    with tempfile.NamedTemporaryFile(mode="w+", delete=False) as temp_file:
+        mlrun.common.constants.MLRUN_SERVING_SPEC_PATH = temp_file.name
+
+
+# Global teardown function
+def tearDownModule():
```

Comment:
Redundant comment that's inconsistent with `setUpModule()`.
```suggestion
def tearDownModule():
```